### ISSUE TITLE
ConcurrentModificationException thrown from CloudRunProcessHandler

### ISSUE THREAD

**Issue by peterlin741** [GoogleCloudPlatform/cloud-code-intellij#2792]
- IntelliJ 2020.1.3
- Cloud Code version: 20.6.1
- OS: Windows 10

```
java.util.ConcurrentModificationException
	at java.base/java.util.ArrayList$Itr.checkForComodification(ArrayList.java:1042)
	at java.base/java.util.ArrayList$Itr.next(ArrayList.java:996)
	at com.google.cloud.tools.intellij.cloudrun.CloudRunProcessHandler$destroyProcess$1$1.invokeSuspend(CloudRunProcessHandler.kt:231)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:740)
```

**Comment by etanshaul** [comment]
bumping this up to Feb

**Comment by ivanporty** [comment]
Fixed in 20.10.1+ versions.
